include/tsl/hopscotch_hash.h

```diff
@@ -1045,14 +1045,14 @@ class hopscotch_hash : private Hash, private KeyEqual, private GrowthPolicy {
   typename U::value_type& operator[](K&& key) {
     using T = typename U::value_type;
 
-    const std::size_t hash = hash_key(key);
-    const std::size_t ibucket_for_hash = bucket_for_hash(hash);
+    const std::size_t __m_hash = hash_key(key);
```

Comment:
Why `__m_hash` here? Double underscores are reserved and I don't see the reason for the `m_` too. 

Comment valid for the other `__m_hash` occurrences in the PR.